pkgs/misc/emulators/retroarch/kodi-advanced-launchers.nix

```diff
@@ -1,9 +1,7 @@
-{ stdenv, pkgs, cores, runtimeShell }:
+{ stdenv, pkgs, lib, cores, runtimeShell }:
 
 assert cores != [];
 
-with pkgs.lib;
```

Comment:
You are sure we do not need ``with lib;`` here anymore?

Reply:
Yes, all of the following references are referencing names that are visibly in scope.

I tested locally:
* before this PR, it fails with `error: undefined variable 'lib' at /home/scott/Projects/nixpkgs/pkgs/misc/emulators/retroarch/kodi-advanced-launchers.nix:38:15`
* after this PR, it evaluates and builds successfully